src/condor_tools/preen.cpp

```diff
@@ -393,21 +393,24 @@ check_spool_dir()
 		"Accountantnew.log",
 		"local_univ_execute",
 		"EventdShutdownRate.log",
-		"OfflineLog",
+		"*OfflineLog*",
 		// SCHEDD.lock: High availability lock file.  Current
 		// manual recommends putting it in the spool, so avoid it.
 		"SCHEDD.lock",
 		"lost+found",
 		};
-	for (int ix = 0; ix < (int)(sizeof(valid_list)/sizeof(valid_list[0])); ++ix) {
-		if ( ! well_known_list.contains(valid_list[ix])) well_known_list.append(valid_list[ix]);
+
+	for (auto & ix : valid_list) {
+		if ( ! well_known_list.contains_withwildcard(ix)) { 
```

Comment:
```suggestion
		if ( ! well_known_list.contains(ix)) { 
```
We're not checking filenames against the list here. That's below, which already uses contains_withwildcard().